.gitattributes

```
@@ -1 +1,2 @@
 *.ipynb linguist-detectable=false
+*.svg binary
```

I thought this would help reduce the diff for the SVG files. Perhaps this needs to be on `main` first?

---

Maybe this will work (based on https://stackoverflow.com/a/44389165) ?
```suggestion
*.svg -text diff
```

---

My understanding is that it is equivalent, but worth a try... tested here https://github.com/banesullivan/test-svg-diff/pull/1 and seems to make no difference

---

ok thanks for checking, seems odd that it's not really working as expected... 🤷‍♂️